### PR TITLE
doc: update base64_decode notes

### PR DESCRIPTION
**Pull request by inashivb**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/5223

### CODE REVIEW

File: doc/userguide/rules/base64-keywords.rst [hunk 1]
```diff
@@ -27,6 +27,10 @@ This option makes ``offset`` skip bytes relative to the previous match.
 
     The content match that you want to decode relative to must be the first match in the stream.
 
+.. note:: ``base64_decode`` follows RFC 4648 by default i.e. encounter with any character that is not found in the base64 alphabet leads to rejection of that character and the rest of the string.
```
Comment (inashivb) [inline doc/userguide/rules/base64-keywords.rst:30]: Does it make sense to have this info as a note @jufajardini ?

Comment (jufajardini) [reply doc/userguide/rules/base64-keywords.rst:30]: I had thought of something like that, indeed!

**Review by jufajardini — APPROVED** [review]
Looks good to me, thanks :)

**Comment by victorjulien** [comment]
Merged in #8042, thanks!

**Pull request closed without merging**